### DISCUSSION TITLE
VIM Resolution mp4 format?

### DISCUSSION THREAD

**Discussion by RadonHD** [Mr-MIBonk/M.I.B._More-Incredible-Bash#222] [General]
Hi there, I allready installed MIB and VIM at 199km/h on my Porsche 718 Boxster. I was able to play a converted video like discribed in this video: https://www.youtube.com/watch?v=3LCW3ZYkEEA with handbreak and the json file linked in youtube but the resolution is not native (native 800x480) in file 360p and only 900kbps. How can I convert files to mp4 with best video settings for my MHI2? Does anyone know where are the caps? I tried 800x480 manually but that was not readable by my MHI2...

**Comment by slee1812alt** [comment]
The video you linked has been removed so I can't see how they are recoding the video.  However, the highest resolution I could get to play on my 718 is 720 x 576. Hope this helps.

**Comment by baazjones** [comment]
Info here works, just use your browsers translate function, It works, the format is 576p
https://auto-prestavby.cz/2020/11/01/mib2-mib2-5-columbus-format-videa/
